roles/bitcurator/tasks/main.yml

```diff
@@ -0,0 +1,146 @@
+---
+# tasks file for roles/bitcurator
+    - name: Update apt cache
+      ansible.builtin.apt:
+        update_cache: true
+        cache_valid_time: 3600
+
+    - name: Create bcadmin user
+      ansible.builtin.user:
+        name: "{{ admin_user }}"
+        state: present
+        createhome: true
+        shell: /bin/bash
+        groups: sudo
+        append: true
+
+    - name: Set password for bcadmin user (optional - change or remove as needed)
+      ansible.builtin.user:
+        name: "{{ admin_user }}"
+        password: "{{ admin_user_password }}"
+
+    - name: Create apps directory
+      ansible.builtin.file:
+        path: "{{ apps_dir }}"
+        state: directory
+        owner: "{{ admin_user }}"
+        group: "{{ admin_user }}"
+        mode: '0755'
+
+    - name: Install BitCurator dependencies
+      ansible.builtin.apt:
+        name: "{{ item }}"
+        state: present
+      loop: "{{ bitcurator_dependencies }}"
+
+    - name: Download BitCurator CLI
+      ansible.builtin.get_url:
+        url: "https://github.com/BitCurator/bitcurator-cli/releases/download/{{ bitcurator_version }}/bitcurator-cli-linux"
+        dest: "{{ download_dir }}/bitcurator-cli-linux"
+        mode: '0644'
+        owner: "{{ admin_user }}"
+        group: "{{ admin_user }}"
+
+    - name: Move BitCurator to /usr/local/bin
+      ansible.builtin.copy:
+        src: "{{ download_dir }}/bitcurator-cli-linux"
+        dest: /usr/local/bin/bitcurator
+        remote_src: true
+        mode: '0755'
+        owner: root
+        group: root
+
+    - name: Run BitCurator install
+      ansible.builtin.command: bitcurator install
+      args:
+        creates: /usr/local/bin/bitcurator
+      register: bitcurator_install
+
+    - name: Download xRDP installer
```

Comment:
Maybe we can emulate this pre-existing role? https://github.com/robertdebock/ansible-role-xrdp